src/cookies.test.ts

```diff
@@ -66,8 +70,36 @@ describe('cookies', () => {
 		);
 	});
 
-	it('should be able to set a session cookie', () => {
+	it('set a cookie for a specific number of days with an expiry that preserves UTC time', () => {
+		// BST started Sun 28th Mar 2021
+		MockDate.set('Sat Mar 27 2021 12:00:00 GMT+0000 (Greenwich Mean Time)');
+		cookies.setCookie({
+			name: 'cookie-1-name',
+			value: 'cookie-1-value',
+			daysToLive: 7,
+		});
+		expect(document.cookie).toEqual(
+			'cookie-1-name=cookie-1-value; path=/; expires=Sat, 03 Apr 2021 12:00:00 GMT; domain=.theguardian.com',
+		);
+	});
+
+	it('does not set a cookie when the cookie name is invalid', () => {
+		cookies.setCookie({
+			name: 'cookie-1-name-@',
+			value: 'cookie-1-value',
+		});
 		expect(document.cookie).toEqual('');
+	});
+
+	it('does not set a cookie when the cookie value is invalid', () => {
```

Comment:
should it throw in this instance to you let you know?

Reply:
Good suggestion. I'll add in another PR so this one remains focussed on the UTC bug fix.